core/src/test/scala/integration/kafka/api/AdminClientTest.scala

```diff
@@ -106,7 +106,7 @@ class AdminClientTest extends IntegrationTestHarness with Logging {
 
     val consumerSummaries = client.describeConsumerGroup(groupId)
     assertEquals(1, consumerSummaries.size)
-    assertEquals(Set(tp, tp2), consumerSummaries.head.assignment.toSet)
+    assertEquals(Set(tp, tp2), consumerSummaries.get.head.assignment.toSet)
```

Comment:
@hachikuji Did you mean just add a function like this - testDescribeConsumerGroupForDeadGroup() or to add a new test file for the ConsumerGroupCommand.scala file altogether? I think the testDescribeConsumerGroupForNonExistentGroup() function checks the dead group scenario and am unsure what you meant exactly.